packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/default.parser.ts

```diff
@@ -285,6 +286,11 @@ class RowProcessor {
       if (typeof this.row[field] === "string") {
         // remove whitespace
         this.row[field] = this.row[field].trim();
+
+        // replace any strings that consist only of whitespace with the empty string
+        if (!this.row[field].match(/\S/)) {
```

Comment:
(non-blocking): It took me a second to understand this, so you are checking for the case where there is not a single non-whitespace character. It's smart and should work as intended, but the double negative throws me a bit. I might be more tempted to use the regex
```
/^[\s]*$/
```
Where 
`^` indicates the start of the string
`$` indicates the end of the string
`[\s]*` matches case when only whitespace provided

Interactive demo for testing at: https://regexr.com/85b97

But I don't feel too strongly about this, more just for interest

Reply:
I am happy to work on it, when I am free too. I am not too confident about the regex myself when I wrote it, so if I can improve it, I am happy to.

Reply:
The regex you wrote does make sense to me and I think should cover all use-cases successfully (appreciate the tests you added to check) so I don't think any additional work specifically required, more just an observation if interested to replace the double-negative with a positive match statement. Regex is quite fiddly at the best of times, I definitely recommend using the regexr.com website linked above for an interactive testing playground

Reply:
I did push one small commit with some comments just as a reminder it would be good to add more tests generally to the parser in the future, so if you ever do find yourself working again on it and are interested that could also be something to look into